packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1271,6 +1351,18 @@ class TextInputConnection {
     );
   }
 
+  /// Send selection rects.
+  ///
+  /// These are used by the engine during a UIDirectScribbleInteraction.
+  void setSelectionRects(List<SelectionRect> selectionRects) {
+    if (!listEquals(_cachedSelectionRects, selectionRects)) {
+      _cachedSelectionRects = selectionRects;
+      TextInput._instance._setSelectionRects(selectionRects.map((SelectionRect rect) {
+        return <num>[rect.bounds.left, rect.bounds.top, rect.bounds.width, rect.bounds.height, rect.position];
```

Comment:
Could `num` be `double` to be more specific?

Reply:
We could probably make it work, but `rect.position` is an `int`, while the bounds properties are `double`, so I think `num` makes the most sense